packages/example-app-base/src/components/templates/PageHeader.tsx

```diff
@@ -63,7 +63,7 @@ export class PageHeaderBase extends BaseComponent<IPageHeaderProps, {}> {
  */
 export const PageHeader = styled(
   // export const PageHeader = createStyled(
-  PageHeaderBase as React.ComponentClass<IPageHeaderProps>,
+  PageHeaderBase as React.ComponentClass<IPropsWithStyles<{}, {}>>,
```

Comment:
This is questionable here, but I'm guessing it is because IPageHeaderProps is a weak type.